src/pages/verified.tsx

```diff
@@ -0,0 +1,53 @@
+import { Button, Center, Title, useMantineTheme } from '@mantine/core';
+import { useMediaQuery } from '@mantine/hooks';
+import { NextLink } from '@mantine/next';
+import { GetServerSideProps, GetServerSidePropsContext } from 'next';
+import { useRouter } from 'next/router';
+import { getServerAuthSession } from '../server/common/get-server-auth-session';
+import { trpc } from '../utils/trpc';
+
+const VerifyRequest = () => {
+  const theme = useMantineTheme();
+
+  const onMobile = useMediaQuery(`(max-width: ${theme.breakpoints.md}px)`);
+  return (
```

Comment:
It's confusing that you have two components called `VerifyRequest`, especially since one of the files is called `verified`. I would suggest changing one of the names.

Reply:
Changed export comp name to Verified